pkg/bootstrap/manifests/klusterlet/clusterrole_priority_class.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: klusterlet-priority-class
+  labels:
+    open-cluster-management.io/aggregate-to-work: "true"
+rules:
+- apiGroups: ["scheduling.k8s.io"]
+  resources: ["priorityclasses"]
+  resourceNames: ["klusterlet-critical"]
```

Comment:
we may need to remove the resourceNames, see the k8s doc [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources):
> Note: You cannot restrict create or deletecollection requests by their resource name. For create, this limitation is because the name of the new object may not be known at authorization time.

Reply:
Updated